Say Their Names/Source/Frameworks/FirebaseIntegration.swift

```diff
@@ -34,20 +33,45 @@ final class FirebaseIntegration {
     // https://firebase.google.com/docs/firestore/query-data/get-data
     
     func getPeople(completion: @escaping (Result<[Person], Error>) -> Void) {
-        db.collection("people").getDocuments { (snapshot, error) in
+        let peopleRef = db.collection("people")
```

Comment:
If we go the route of Firebase, can we make the references instance properties of `FirebaseIntegration`?

Reply:
Sure, that’s a nice idea!